pkg/types/installconfig.go

```diff
@@ -351,3 +351,87 @@ type BootstrapInPlace struct {
 	// InstallationDisk is the target disk drive for coreos-installer
 	InstallationDisk string `json:"installationDisk"`
 }
+
+// ClusterCreateError defines a custom error type that would help identify where the error occurs
+// during the bootstrap phase of the installation process. This would help identify whether the error
+// comes either from the Kubernetes API failure, the bootstrap failure or a general kubernetes client
+// creation error. In the event of any error, this interface packages the error message and a custom
+// log message that must be neatly presented to the user before termination of the project.
+type ClusterCreateError interface {
+	GetError() error
```

Comment:
Use `Unwrap` so that the `ClusterCreateError` can be treated as a wrapped error.
```suggestion
	Unwrap() error
```